app/views/schools/new.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1 class="govuk-heading-xl">New school</h1>
+
+<%= form_with model: @school ||= School.new, url: schools_path, local: true do |form| %>
+  <% if @school&.errors&.any? %>
+    <%= form.govuk_error_summary %>
+  <% end %>
+  <%= form.govuk_text_field :name %>
+
+  <%= form.govuk_date_field :opened,
+                            legend: { text: "Enter the date the school was opened" },
+                            hint: { text: "For example, 31 3 1980" }
+  %>
+
+  <%= form.govuk_collection_radio_buttons :school_type,
+                                          School::SCHOOL_TYPES,
+                                          ->(option) { option },
+                                          ->(option) { option },
+                                          legend: { text: "What type of school is it?" }
+  %>
```

Comment:
Something we should discuss and stick to: What whitespace format do we want? 
1. Long single-line
2. Multiline (like here)
3. Multiline with parentheses: 
```
  <%= form.govuk_collection_radio_buttons(
        :school_type, 
        School::SCHOOL_TYPES,
        ->(option) { option },
        ->(option) { option },
        legend: { text: "What type of school is it?" }
      )
  %>
```
We should probably decide which multiline to use (sometimes single line makes sense imo), and stick to it. I think I like 3 more than 2, because it prevents indentation depending on length of form builder method we want to use.